platforms/bukkit/common/build.gradle.kts

```diff
@@ -4,15 +4,12 @@ repositories {
 
 dependencies {
     shadedApi(project(":common:implementation:base"))
-    
-    api("org.slf4j:slf4j-api:1.8.0-beta4") {
```

Comment:
Are you sure that we no longer need to shade this? Has 6.4.0 been tested on older versions of paper? (eg. mc version 1.18)

Reply:
the comment said, "Minecraft 1.17+ includes slf4j 1.8.0-beta4, so we need to shade it for other versions." I assume the comment was down with actual knowledge that it was added. but no it has not been tested paging @justaureus to test :)

Reply:
We no longer support 1.18, do you mean 1.19?

Reply:
we support 1.18.2 on bukkit

Reply:
Doesn't appear we do [here](https://terra.polydev.org/install/versions.html#bukkit).

Reply:
1.18.2 is still in tree

Reply:
![image](https://github.com/PolyhedralDev/Terra/assets/25417900/7a203abe-558d-427f-b41e-cd68a85840fd)

Reply:
![image](https://github.com/PolyhedralDev/Terra/assets/25417900/515605c1-bf12-4f1f-993a-9251e99beec2)

Reply:
Okay, gotcha.  Will test then.

Reply:
> the comment said, "Minecraft 1.17+ includes slf4j 1.8.0-beta4, so we need to shade it for other versions." I assume the comment was written down with actual knowledge that it was added. but no it has not been tested paging @justaureus to test :)

when I wrote it, it *may* have been because versions of bukkit after 1.17 depended on a non-beta version of slf4j, but at runtime used a newer version. I could be incorrect, though.